modules.d/01fips/module-setup.sh

```diff
@@ -64,7 +64,7 @@ install() {
     inst_hook pre-udev 01 "$moddir/fips-load-crypto.sh"
     inst_script "$moddir/fips.sh" /sbin/fips.sh
 
-    inst_multiple sha512hmac rmmod insmod mount uname umount grep sed cut find sort
+    inst_multiple sha512hmac rmmod insmod mount uname umount grep sed cut find sort tail tr
```

Comment:
Shouldn't you also add `cat` here, just to make sure it actually exists?

Reply:
Nice catch, will add!